server/src/main/java/io/spine/server/tuple/Either.java

```diff
@@ -102,17 +99,16 @@ public final boolean equals(Object obj) {
         if (this == obj) {
             return true;
         }
-        if (!(obj instanceof Either)) {
+        if (!(obj instanceof Either other)) {
```

Comment:
This one makes no sense to me. I would expect IDEA to file a warning here.

Reply:
This is new Java feature called "pattern variable".

Reply:
IDEA suggested changing the code this way. So, I did to avoid the warnings :)